discover/discover.go

```diff
@@ -149,11 +182,13 @@ func ConvertPolicy(c *k8s.Client, o Options) error {
 // Policy discovers Cilium or KubeArmor policies
 func Policy(c *k8s.Client, o Options) error {
 	if o.Policy == "cilium" {
-		o.Policy = "network"
+		o.Policy = "network-cilium"
 	} else if o.Policy == "kubearmor" {
 		o.Policy = "system"
+	} else if o.Policy == "k8snetpol" {
+		o.Policy = "network-generic"
```

Comment:
Can we use the `kind` in place of using our own term?
```suggestion
		o.Policy = "NetworkPolicy"
```